feat(casino): add new steps and bonus time (#56)
QCGSecret/website: src/pages/guide/casino.mdx

</Dropdown>

<Header title="Chosen" id="chosen">

> 	1 player

Reviewer: ```suggestion
	1-3 players
```

> 	1 players

Reviewer: Pas besoin de s
```suggestion
	1 player
```

> 	1 players

Reviewer: ```suggestion
	1-3 players
```

</Header>

After assembling the ATM code, you can insert the credit card. You will then be prompted to enter a PIN code, but be careful—you only have three attempts. 

> Once the correct PIN is entered, wait a few seconds for the money to be dispensed. Afterward, collect the money and place it in the coin exchanger to receive two yellow coins and one green coin.

Reviewer: Préciser le temps ?

> One person must collect the green coin and use it on a gambling machine. After a short animation showing a win, the player will receive a lucky message. Following this, interact with the fates machine; unlike before, rather than perishing, you will receive an electric shock and be granted the Chosen Title. Go outside and approach the arcade machine to shock and activate it.

Reviewer: Pas besoin de s
```suggestion
One person must collect the green coin and use it on a gambling machine. After a short animation showing a win, the player will receive a lucky message. Following this, interact with the fate machine; unlike before, rather than perishing, you will receive an electric shock and be granted the Chosen Title. Go outside and approach the arcade machine to shock and activate it.
```

<Header title="Arcade" id="arcade">
	8 players
</Header>

> After activating the machine, you'll need to insert 8 coins. Following that, you'll be prompted to donate blood; 8 people must interact with the blade on the machine and press the green buttons to initiate the game. Be cautious, as interacting with it will set your HP to 1.

Reviewer: Pas besoin de s et plus de précision
```suggestion
After activating the machine, you'll need to insert 8 coins. Following that, you'll be prompted to donate blood; 8 people must interact with the blade on the machine and press the green button to initiate the game. Be cautious, as interacting with the blade will set your HP to 1.
```

> This steps gives a 120 seconds time bonus when completed.

Reviewer: Formatting
```suggestion
This steps gives a **_120_** seconds time bonus when completed.
```
